tests/plugins/test_mutual_inclusion.py

```diff
@@ -0,0 +1,40 @@
+"""Provides Unit Tests for the `abis_mapping.plugins.mutual_inclusion` module"""
+
+
+# Third-Party
+import frictionless
+
+# Local
+from abis_mapping import plugins
+
+
+def test_checks_mutually_inclusive() -> None:
+    """Tests the MutuallyInclusive Checker"""
+    # Construct Fake Resource
+    resource = frictionless.Resource(
+        source=[
+            # Valid
+            {"a": "A", "b": "B", "c": "C"},
+            {"a": "A", "b": "B", "c": None},
+            {"a": None, "b": None, "c": "C"},
+
+            # Invalid
+            {"a": None, "b": "B", "c": "C"},
+            {"a": None, "b": "B", "c": None},
+            {"a": "A", "b": None, "c": "C"},
+            {"a": "A", "b": None, "c": None},
+        ],
+    )
+
+    # Validate
+    report: frictionless.Report = resource.validate(
+        checks=[
+            plugins.mutual_inclusion.MutuallyInclusive(
+                field_names=["a", "b"],
+            ),
+        ]
+    )
+
+    # Check
+    assert not report.valid
+    assert len(report.flatten()) == 4
```

Comment:
A general remark for all the `Check` tests: it would be better if we could assert which row  is invalid instead of just the count of invalid rows. I don't know if the report allows you to have access at the row number.
Not important for now.